src/review/Post.js

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useState } from 'react';
+import Nav from '../../src/components/Nav/Nav';
+import { AiOutlineArrowUp, AiOutlineArrowDown } from 'react-icons/ai';
+import TextItem from '../components/review/TextItem';
+import './Post.scss';
+
+const PRODUCT_INFO = [
+  {
+    img: 'https://crowcanyon.co.kr/web/product/tiny/202201/340ff70c75de7d1e91b0979a59a228e6.jpg',
+    NEW: true,
+    품절: true,
+    name: 'D49 스몰스푼 베이비 블루마블',
+    price: 9000,
+  },
+];
+
+const Post = () => {
+  const [textList, setTextList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/relatedProductData.json')
+      .then(res => res.json())
+      .then(productData => setTextList(productData));
+  }, []);
+
+  return (
+    <>
+      <Nav />
+      <div className="Post">
```

Comment:
```suggestion
      <div className="post">
```
className은 camelCase 지켜주세요

Reply:
수정했습니다

Reply:
가장 바깥 div는 컴포넌트 이름과 같아야 하는 게 아닌가요? ㅠㅠ 수정하니 오류가 나서 일단 다시 돌렸습니다

Reply:
컴포넌트 이름과 같되 camelCase 지켜서 작성하도록 가이드 드렸습니다! className을 변경하실 때는 scss도 수정해야지 스타일이 적용되겠죠? 아마 그부분 때문에 오류가 났다고 하신 것 같네요

Reply:
대소문자로 구분된 post가 존재해서 그랬습니다 ㅠ 수정했습니다